lighthouse-core/gather/driver/navigation.js

```diff
@@ -58,7 +79,7 @@ function resolveWaitForFullyLoadedOptions(options) {
  * @param {LH.Gatherer.FRTransitionalDriver} driver
  * @param {string} url
  * @param {NavigationOptions} options
- * @return {Promise<{finalUrl: string, timedOut: boolean}>}
+ * @return {Promise<{finalUrl: string, timedOut: boolean, warnings: Array<LH.IcuMessage>}>}
```

Comment:
do we still need `timedOut`?

Reply:
not to accomplish today's functionality, no. prefer removal?

Reply:
ya